npm/npm.go

```diff
@@ -83,15 +81,10 @@ func NewNetworkPolicyManager(informerFactory informers.SharedInformerFactory, ex
 		npInformer:        informerFactory.Networking().V1().NetworkPolicies(),
 		ipsMgr:            ipsm.NewIpsetManager(exec),
 		npmNamespaceCache: &npmNamespaceCache{nsMap: make(map[string]*Namespace)},
-		clusterState: telemetry.ClusterState{
-			PodCount:      0,
-			NsCount:       0,
-			NwPolicyCount: 0,
-		},
-		k8sServerVersion: k8sServerVersion,
-		NodeName:         os.Getenv("HOSTNAME"),
-		version:          npmVersion,
-		TelemetryEnabled: true,
+		k8sServerVersion:  k8sServerVersion,
+		NodeName:          os.Getenv("HOSTNAME"),
```

Comment:
can you also add this(nodeName) for cni future PR if not present

Reply:
ignore.. AI wrapper package adds vm name

Reply:
@tamilmani1989  I guess you meant in [CNIReport](https://github.com/Azure/azure-container-networking/blob/master/telemetry/telemetry.go#L79) struct?

Reply:
ignore my comment.. nodename will get added by default in AItelemetry package